Patch originally from https://github.com/oracle/solaris-userland/

PIP on OmniOS should not use its bundled cacert.pem file, it should instead
use system certificates file which is more likely to be updated in a timely
manner when a CA cert is added/removed/modified.

diff -wpruN '--exclude=*.orig' a~/src/pip/_vendor/certifi/core.py a/src/pip/_vendor/certifi/core.py
--- a~/src/pip/_vendor/certifi/core.py	1970-01-01 00:00:00
+++ a/src/pip/_vendor/certifi/core.py	1970-01-01 00:00:00
@@ -23,34 +23,8 @@ try:
             return _PIP_STANDALONE_CERT
         raise _PipPatchedCertificate()
 
-    from importlib.resources import path as get_path, read_text
-
-    _CACERT_CTX = None
-    _CACERT_PATH = None
-
-    def where():
-        # This is slightly terrible, but we want to delay extracting the file
-        # in cases where we're inside of a zipimport situation until someone
-        # actually calls where(), but we don't want to re-extract the file
-        # on every call of where(), so we'll do it once then store it in a
-        # global variable.
-        global _CACERT_CTX
-        global _CACERT_PATH
-        if _CACERT_PATH is None:
-            # This is slightly janky, the importlib.resources API wants you to
-            # manage the cleanup of this file, so it doesn't actually return a
-            # path, it returns a context manager that will give you the path
-            # when you enter it and will do any cleanup when you leave it. In
-            # the common case of not needing a temporary file, it will just
-            # return the file system location and the __exit__() is a no-op.
-            #
-            # We also have to hold onto the actual context manager, because
-            # it will do the cleanup whenever it gets garbage collected, so
-            # we will also store that at the global level as well.
-            _CACERT_CTX = get_path("pip._vendor.certifi", "cacert.pem")
-            _CACERT_PATH = str(_CACERT_CTX.__enter__())
-
-        return _CACERT_PATH
+    def where(): # keepindent
+        return "/etc/ssl/cacert.pem"
 
 except _PipPatchedCertificate:
     pass
@@ -73,4 +47,5 @@ except ImportError:
 
 
 def contents():
-    return read_text("certifi", "cacert.pem", encoding="ascii")
+    with open(where(), "r", encoding="ascii") as data:
+        return data.read()
